luigi_pipeline/seqr_mito_loading.py

```diff
@@ -35,9 +36,37 @@ def annotate_globals(self, mt):
 
 
 class SeqrMitoGenotypesMTTask(BaseVCFToGenotypesMTTask):
+    ignore_missing_samples = luigi.BoolParameter(default=False, description='Allow missing samples in the callset.')
     VariantTask = SeqrMitoVariantMTTask
     GenotypesSchema = SeqrMitoGenotypesSchema
 
+    def subset_samples_and_variants(self, mt, subset_path):
+        """
+        Override the base class method to allow missing samples for mitochondria datasets
+        """
+        subset_ht = hl.import_table(subset_path, key='s')
+        subset_count = subset_ht.count()
+        anti_join_ht = subset_ht.anti_join(mt.cols())
+        anti_join_ht_count = anti_join_ht.count()
+
+        if anti_join_ht_count != 0:
+            missing_samples = anti_join_ht.s.collect()
+            message = f'Only {subset_count - anti_join_ht_count} out of {subset_count} ' \
+                      f'subsetting-table IDs matched IDs in the variant callset.\n' \
+                      f'IDs that aren\'t in the callset: {missing_samples}\n' \
+                      f'All callset sample IDs:{mt.s.collect()}'
+            if (subset_count > anti_join_ht_count) and self.ignore_missing_samples:
```

Comment:
this is almost entirely copy-pasted from the base function, this line is basically the only part that changed. When I said you needed to override it, I meant you should override the very small part that is different, not just copy-paste it. Shared behavior needs to actually be shared, when you copy it out it makes future maitenance basically impossible. You can achieve this by adding a keyword argument to the base class and then use super to set that keyword properly:
```
def subset_samples_and_variants(self, *args):
    return super().subset_samples_and_variants(*args, ignore_missing_samples=self.ignore_missing_samples)
```

Reply:
yes and we change the base class method to take a keyword arg where the default behavior is the current behavior